packages/angular_devkit/benchmark/src/main_spec.ts

```diff
@@ -142,4 +143,13 @@ describe('benchmark binary', () => {
     stdout.lines.forEach(line => expect(line).toMatch(/^\[abc\]/));
     expect(res).toEqual(0);
   });
+
+  it('uses watch-script and watch-matcher', async () => {
```

Comment:
Can you add successful and failing retry tests, and timeout tests as well?

Checking the verbose output helps to make sure the right thing is happening.